.circleci/restore_build_output.sh

```diff
@@ -32,6 +32,13 @@ if [[ -d "${WORKSPACE_DIR}/builds" ]]; then
       mkdir -p $RESTORED_DIR
       rsync -a "${RESTORED_COMPONENT_DIR}/" "./${OUTPUT_DIR}"
     done
+    # Bento components are compiled inside the src/bento source file.
```

Comment:
Wait so which statement is true? This one or the previous comment?
If it's both, let's reword them so they don't sound like they conflict with each other

Reply:
Both are true.